### ISSUE TITLE
Domain issue.

### ISSUE THREAD

**Issue by Hontu** [Hontu/cssforge#1]
The website "http://cssforge.io" doesn't exist and we need someone to buy the domain.
We will allow anyone to purchase it. Please purchase the domain using these hosting services.
> GoDaddy
> Freenom
> HostGator
> Bluehost

**Comment by Purifiy** [comment]
I agree. "http://cssforge.io/" is available, but the problem is the cost on > GoDaddy.
It's 50 dollars. If no one has enough money for that we can get "http://cssforge.net/".

**Comment by Hontu** [comment]
> I agree. "http://cssforge.io/" is available, but the problem is the cost on > GoDaddy.
> It's 50 dollars. If no one has enough money for that we can get "http://cssforge.net/".

I agree.

**Comment by PacksGamingHD** [comment]
Get cssforge.net

When you get big and get money get the .io and get a redirect on the .net

**Comment by Hontu** [comment]
> Get cssforge.net
> When you get big and get money get the .io and get a redirect on the .net

That's a great idea.

**Comment by Purifiy** [comment]
I agree with the idea. @Hontu, move cssforge to an organization.

**Comment by Purifiy** [comment]
> > Get cssforge.net
> > When you get big and get money get the .io and get a redirect on the .net

> That's a great idea. 

Agreed with @Hontu and @PacksGamingHD.

**Comment by falceso** [comment]
For the redirect I would use Permanent  Redirect 301

**Comment by Ragweed** [comment]
> For the redirect I would use Permanent  Redirect 301
I have a no idea what that is but sounds like a great idea :D !
